### PR TITLE
Commenting

### PR DESCRIPTION
**Pull request by gjtorikian**
This is an example of a Pull Request.

### CODE REVIEW

**Comment by gjtorikian** [comment]
This is an example of a comment on an entire Pull Request.

File: fake_file.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Hey, _here's_ some neat content!
+
+Plus, I can just add lines.
```
Comment (gjtorikian) [inline fake_file.md:3]: Here's a comment on a specific line in a Pull Request.

**Comment by tkrausjr** [comment]
Testing for a specific #

**Comment by ivanromakh** [comment]
wewe

File: fake_file.md [hunk 2]
```diff
@@ -1 +1,3 @@
-hey here's some neat content!
\ No newline at end of file
+Hey, _here's_ some neat content!
```
Comment (sarasafavi) [inline fake_file.md]: test comment pls ignore

Comment (sarasafavi) [inline fake_file.md]: specific line in a specific commit, single comment

File: fake_file.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
 Hey, _here's_ some neat content!
 
 Plus, I can just add lines.
+
+
+How far should I do?
```
Comment (sarasafavi) [inline fake_file.md]: Comment on L6 in commit cbc28e7

Comment (sarasafavi) [inline fake_file.md]: Making the above, I had the option to "Add single comment" or "Start a review" (chose the former)

File: fake_file.md [hunk 4]
```diff
@@ -1 +1,3 @@
-hey here's some neat content!
\ No newline at end of file
+Hey, _here's_ some neat content!
+
```
Comment (sarasafavi) [inline fake_file.md]: Comment on L2 in 60a2fec

File: fake_file.md (hunk 1 above)
Comment (harshalmittal4) [reply fake_file.md:3]: Just testing!

File: fake_file.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+Plus, I can just add lines.
+
+
+How far should I do?
```
Comment (geppy) [inline fake_file.md:6]:
```suggestion
How far should I go?
```

**Comment by errfanwadia** [comment]
asdad

**Comment by Alexmao12** [comment]
I am good

- [ ] kill a bird
- [ ] eat it
- [ ] ran away

1. test
2. test 
[Google](Google.com)

**Comment by Alexmao12** [comment]
<h2>Real-time online HTML editor</h2><p>Create and edit <a href="https://validator.w3.org/" rel="nofollow" target="_blank" title="W3C HTML validator">W3C valid HTML code</a> without writing a single line of code. The WYSIWYG editor on the left helps non-developers write their own <em>high-quality HTML code</em>.</p><h3>Know some code? <span class="fr-emoticon fr-deletable fr-emoticon-img" style="background: url(https://cdnjs.cloudflare.com/ajax/libs/emojione/2.0.1/assets/svg/1f600.svg);">&nbsp;</span></h3><p>The editing works both ways, with live visual feedback. Write some markup code on the right, and it will appear in the left editor.</p><p><br></p><table style="width: 100%;"><tbody><tr><td style="width: 50.0000%;">Good</td><td style="width: 50.0000%;">bad</td></tr><tr><td style="width: 50.0000%;"><br></td><td style="width: 50.0000%;"><br></td></tr></tbody></table>

**Comment by Alexmao12** [comment]
## Oops ! :ghost: Not quite there yet !

It seems you have failed some of our test. Details contained in the following comment, please remediate before carry on. 

| Overall               | Fail               |
| --------------------- | ------------------ |
| Supply Chain Analysis | [Fail](google.com) |
| Code scan             | [Fail](Google.com) |
| Secret scan           | [Pass](Google.com) |

We check your code against [SKY application security standards](google.com), Thank you very much for safeguard our users. --SKY Group Software Security  [Slack](https://app.slack.com/client/T02GEFU92/G01FZ201X0W)

**Comment by Alexmao12** [comment]
#24058

**Comment by kami722** [comment]
The script is not working in my Photoshop cc2015 and  I don't know how it can  e done. and How it can be fix

**Comment by islamseeq** [comment]
A deleted comment

File: fake_file.md (hunk 1 above)
Comment (ArunAtGitHubb) [reply fake_file.md:3]: Also just testing!!

File: fake_file.md (hunk 5 above)
Comment (ArunAtGitHubb) [reply fake_file.md:6]: Is this work in here too???

**Comment by Srustika** [comment]
Is there a way to print build logs onto github comment section of  a Pull Request using github API?

**Comment by vikas-shrivastava-db** [comment]
I am adding a comment.

**Comment by vikas-shrivastava-db** [comment]
Another comment

**Comment by nalbion** [comment]
Can you make comments on files that are not changed by the PR? 

Use case:

You also need to add the library to package.json for your changes:

```
`` suggestion:package.json:32
  "new-lib": "1.0.0",
``
```

File: fake_file.md (hunk 1 above)
Comment (BackendArchitectX) [reply fake_file.md:3]: Just Testing!

Comment (StonerHamster) [reply fake_file.md:3]: Testing

**Comment by BackendArchitectX** [comment]
This is just another comment!

File: fake_file.md (hunk 1 above)
Comment (miahshywoods-ux) [reply fake_file.md:3]: Testing

Comment (miahshywoods-ux) [reply fake_file.md:3]: Testing

**Comment by preethampreethampramodessala-create** [comment]
nina abipraya enu

**Pull request closed without merging**